src/System.Private.CoreLib/src/System/Environment.UWP.cs

```diff
@@ -46,5 +46,11 @@ public static string MachineName
                 throw new PlatformNotSupportedException();
             }
         }
+
+        public static void Exit(int exitCode)
+        {
+            // Store apps have their lifetime managed by the PLM
+            throw new PlatformNotSupportedException();
```

Comment:
I'm not sure if there is a WinRT API to exit the process but if there is that what I would prefer. Otherwise if we can mimic a sane behavior and exit the process then we should do that. FailFast might be a reasonable way to do that but I'm not sure how destructive that is over calling ExitProcess in Win32 for example.